app/assets/scss/application.scss

```diff
@@ -1,7 +1,4 @@
-// GOVUK Front-end toolkit styles
-@import "_grid_layout.scss";
-@import "_typography.scss";
-@import "_url-helpers.scss";
+@import "govuk-elements";
```

Comment:
How do these imports work? Why has it changed?

Reply:
Previously we imported 'some' of govuk-elements. Now we're importing all of it. I can't really see any reason to selectively import (now that we need more than a few files) as it just makes life more complicated, although probably I should have looked at the generated application.css before and afterwards...

Reply:
👍